atomic_reactor/tasks/binary_container_build.py

```diff
@@ -225,6 +227,7 @@ def build_container(
         build_args: Dict[str, str],
         dest_tag: ImageName,
         flatpak: bool,
+        memory_limit: str,
```

Comment:
Shouldn't be this `Optional[str]`?

Reply:
well I am always passing it, so it isn't optional

Reply:
No you are not, this can be `None`:
```python
memory_limit=config.remote_hosts.get("memory_limit"),
```

Reply:
ok fixed